fecgov/fec-cms
# 5734 export all elections

## Summary

- Resolves #5734 

Add the option to export the All Elections data on the [House](http://127.0.0.1:8000/data/elections/house/) and [Senate](http://127.0.0.1:8000/data/elections/senate/) elections pages

### Required reviewers

- UX
- front-end

## Impacted areas of the application

The totals section of the House and Senate elections pages, and anywhere select.two_year_select is used

## Screenshots

The export button should disappear for the smaller sizes, just like the rest of the site
<img width="463" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/b40f714f-7bfb-468f-9db6-427b3899722f">

<img width="1067" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/424ad791-fb09-46fb-87d5-0b6d739a1540">

Updated vertical spacing:
![image](https://github.com/fecgov/fec-cms/assets/26720877/31bff385-a167-4ce9-a657-66e3b20326fb)


## Related PRs

None

## How to test

- pull the branch (or check Feature) (If you're testing locally, the download itself should fail due to a CORS error)
- `npm run build`
- `./manage.py runserver`
- check the House (`/data/elections/house/`) and Senate (`/data/elections/senate/`) elections pages, the Totals section at the bottom
- change the year
- click the export button
- make sure the dialog and download process is working
- make sure the downloaded file is correct
- (on localhost, clicking the export button should open the "Your downloads" window but with an error message)
